shipyard/README.md

```diff
@@ -0,0 +1,60 @@
+# Shipyard
+
+## Overview
+
+Shipyard unblocks DevOps teams from bulky, outdated production pipelines. We resolve the pre-production or staging bottleneck with our agile on-demand ephemeral (testing or preview) environments.
+
+The Datadog Shipyard integration automatically passes your [Shipyard][1] ephemeral environment logs to your Datadog dashboard. For every application you have enables the Datadog logs for, Shipyard sends the runtime logs of your application to your dashboard. 
```

Comment:
```suggestion
The Datadog Shipyard integration automatically passes your [Shipyard][1] ephemeral environment logs to Datadog, for all applications on which you have enabled Datadog logs. 
```
Suggesting to shorten this text a bit, but let me know if this still makes sense (feel free to correct if not)